tests/expectations/metrics/test_core.py

```diff
@@ -77,6 +77,98 @@ def test_basic_metric_pd():
     assert results == {desired_metric.id: 3}
 
 
+@pytest.mark.parametrize(
+    "build_engine,dataframe,expected_result",
+    [
+        [
+            build_pandas_engine,
+            pd.DataFrame({"a": [1, 2, 3, None]}),
+            6,
+        ],
+        [
+            build_pandas_engine,
+            pd.DataFrame({"a": [Decimal(2.0), Decimal(0.18781)]}),
+            2.18781,
+        ],
+    ],
+)
+def test_column_sum_metric_pd(build_engine, dataframe, expected_result):
+    engine = build_engine(df=dataframe)
+
+    metrics: Dict[Tuple[str, str, str], MetricValue] = {}
+
+    table_columns_metric: MetricConfiguration
+    results: Dict[Tuple[str, str, str], MetricValue]
+
+    table_columns_metric, results = get_table_columns_metric(execution_engine=engine)
+    metrics.update(results)
+
+    desired_metric = MetricConfiguration(
+        metric_name="column.sum",
+        metric_domain_kwargs={"column": "a"},
+        metric_value_kwargs=None,
+    )
+    desired_metric.metric_dependencies = {
+        "table.columns": table_columns_metric,
+    }
+    results = engine.resolve_metrics(
+        metrics_to_resolve=(desired_metric,), metrics=metrics
+    )
+    metrics.update(results)
+    assert results == {desired_metric.id: expected_result}
+
+
+@pytest.mark.parametrize(
+    "dataframe,expected_result",
+    [
+        [
+            pd.DataFrame({"a": [1, 2, 3, None]}),
+            6,
+        ],
+        [
+            pd.DataFrame({"a": [Decimal(2.0), Decimal(0.18781)]}),
+            2.18781,
+        ],
+    ],
+)
+def test_column_sum_metric_spark(spark_session, dataframe, expected_result):
```

Comment:
This feels out of scope for placing S3 and Trino under the `Compatiblity` pattern. Is there a reason why this test is being included?

Reply:
This will be merged from another PR (approved).  Thanks.

Reply:
👍🏼